resources/assets/components/FilterBar/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './filter-bar.scss';
+
+import DropdownFilter from '../DropdownFilter';
+import MultiValueFilter from '../MultiValueFilter';
+
+class FilterBar extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      filters: {},
+    };
+
+    // The component provides a function the children components can use to update the state.
+    this.updateFilters = this.updateFilters.bind(this);
+  }
+
+  updateFilters(values) {
```

Comment:
Since you now always pass in a key/value object to this, I think you could just push the full `values` object to the state. i.e the Dropdown changes and pushes `{status: accepted}` so you don't need to check if it is a status, you just push that object to the state.

Reply:
ah yes! Just did some more work to pass the `type` to the `MultiValueFilter` so that the `tags` layer is also included in the state in the `MultiValueFilter` in order for this to work!